10.IDR_metadata/scripts/3.create_library.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[2]:
+
+
+import pathlib
+
+import pandas as pd
+
+# In[3]:
+
+
+# set the ouput path for the library file
+output_path = pathlib.Path("../IDR_metadata/screenA/idr0000-screenA-library.txt")
+
+
+# In[5]:
```

Comment:
These notebooks are not executed sequentially

Reply:
re-ran this notebook to be run sequentially! :D